api/src/main/java/io/grpc/MetricInstrumentRegistry.java

```diff
@@ -17,35 +17,31 @@
 package io.grpc;
 
 import com.google.common.annotations.VisibleForTesting;
-import com.google.common.collect.ImmutableList;
-import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-import java.util.concurrent.atomic.AtomicInteger;
+import javax.annotation.concurrent.GuardedBy;
 
 /**
  * A registry for globally registered metric instruments.
  */
 @Internal
 public final class MetricInstrumentRegistry {
-  private static final int DEFAULT_INSTRUMENT_LIST_CAPACITY = 20;
+  public static final int INITIAL_INSTRUMENT_CAPACITY = 5;
```

Comment:
Don't make this public. It is only used by a test in the same package, so package-private is sufficient.